frontend/src/lib/components/instances/views/elements/Image.svelte

```diff
@@ -0,0 +1,5 @@
+<script lang="ts">
+	export let data: string;
+</script>
+
+<img src={data} style:max-width="200px" alt="Image thumbnail for instance {data}" />
```

Comment:
Can we make things like the max width a parameter? Are there arbitrary parameter lists options we can pass in to views?